doc/sphinx-guides/source/api/changelog.rst

```diff
@@ -7,6 +7,11 @@ This API changelog is experimental and we would love feedback on its usefulness.
     :local:
     :depth: 1
 
+v6.4
+----
+
+- /api/metadatablocks is now returning no duplicated metadata properties and does not ommit metadata properties when called. The JsonPrinter class output is fixed.
```

Comment:
```suggestion
- **/api/metadatablocks** is now returning no duplicated metadata properties and does not omit metadata properties when called.
```

I'm suggesting that we remove the reference to JsonPrinter because API users don't need to worry about the underlying implementation.

Reply:
@ffritze 
The requested change was not made. Also this now has to be added under V6.5 in the changeling.rst

Reply:
Done.